### PR TITLE
Harbor, boats

### PR DESCRIPTION
**Pull request by Dzierzan**
This PR adds custom Harbor building and two basic boats and water splash.

I intend to improve the Harbor building in the future and of course make proper remaps for both boats.

### CODE REVIEW

**Comment by GraionDilach** [comment]
Public Domain is CC0, you don't need a separate folder.

File: mods/hv/rules/buildings.yaml [hunk 1]
```diff
@@ -703,3 +703,65 @@ TELEVTR2:
...
+		Bounds: 40,40
+	Buildable:
+		Prerequisites: base
+		Queue: Building
+		BuildPaletteOrder: 90
+		Description: Builds ships.
+	Valued:
+		Cost: 1000
+	Tooltip:
+		Name: Habor
```
Comment (GraionDilach) [inline mods/hv/rules/buildings.yaml]: HaRbor

File: mods/hv/rules/defaults.yaml [hunk 2]
```diff
@@ -368,3 +368,51 @@
...
+		GenericName: Ship
+	MustBeDestroyed:
+	MapEditorData:
+		Categories: Naval
+	Voiced:
+		VoiceSet: GenericVoice2
+	RenderSprites:
+		PlayerPalette: remap
+	WithFacingSpriteBody:
+	HitShape:
```
Comment (GraionDilach) [inline mods/hv/rules/defaults.yaml]: Missing newline before end of file. OpenRA convention is to end yamls with newlines.

File: mods/hv/sequences/weapons.yaml [hunk 3]
```diff
@@ -84,3 +84,16 @@ flame:
...
+	water_splash_a: water_splash.png
+		Start: 0
+		Length: 9
+		Tick: 60
+		ZOffset: 1000
+	water_splash_b: water_splash.png
+		Start: 9
+		Length: 9
+		Tick: 60
+		ZOffset: 1000
```
Comment (GraionDilach) [inline mods/hv/sequences/weapons.yaml]: Missing newline + additional tabulators.

File: mods/hv/sequences/weapons.yaml [hunk 4]
```diff
@@ -84,3 +84,16 @@ flame:
 	end: flame.png
 		Length: *
 		ZOffset: 512
+
+
```
Comment (GraionDilach) [inline mods/hv/sequences/weapons.yaml]: Superflous newline.

File: mods/hv/rules/player.yaml [hunk 5]
```diff
@@ -48,6 +48,11 @@ Player:
 		DisplayOrder: 4
 		LowPowerModifier: 300
 		SpeedUp: true
+	ClassicProductionQueue@Naval:
+		Type: Ship
+		DisplayOrder: 4
```
Comment (Mailaender) [inline mods/hv/rules/player.yaml]: 5

File: mods/hv/weapons/ballistics.yaml [hunk 6]
```diff
@@ -21,6 +21,12 @@
...
 		ImpactSounds: explosion06.wav
+	Warhead@WaterEffect: CreateEffect
+		Image: water_splash
+		Explosions: water_splash_a, water_splash_b
+		ValidTargets: Water
+		InvalidTargets: Ship, Structure
+		ImpactSounds: Video_Game_Splash-Ploor.wav
 
 TyrianTankCannon:
 	Inherits: ^Cannon
```
Comment (Mailaender) [inline mods/hv/weapons/ballistics.yaml]: Try adding `Water` to the first line:

```
	ValidTargets: Ground, Tree
```

to make it splash on water.

**Pull request merged**